### DISCUSSION TITLE
Where remote DevOps teams excel long-term?

### DISCUSSION THREAD

**Discussion by bojanterzic529** [community/community#30105] [Other Feature Feedback, Questions, & Ideas] [answered]
I am a Web developer.
During the development, some issues were crashed, so let me ask one thing.
Where remote DevOps teams excel long-term?

**Comment by solyasa** [comment, marked as answer]
Remote working is of great benefit to employers and employees both in terms of cost, productivity, and quality of life. What I have found to be very effective is to get the members of the team together for a few days a quarter or so; just a small amount of face time with people helps build the relationships in a way that sticks, and the remote working works even better when the human feels they know the other human, no matter where in the world they may be

**Reply by bojanterzic529** [reply]
thank you
It was great help

**Comment by solyasa** [comment]
Also devOps have many advantage in software development.

Through improving the synergy between the development and operation departments of businesses, companies can help developers understand what problems are being faced, and create easier ways to debug blocks of code through the various DevOps tools.

On the other hand, operation teams can focus on their work, while still making sure that the needs of developers are handled as they come up. This, in turn, helps create a continuous workflow, ensuring fast deployment, and efficient patch updates to remove any bugs.

I want to this article will help your work.

**Reply by bojanterzic529** [reply]
thank you very much

**Comment by Emanuelcarvalho2004** [comment]
Isso é muito interessante e acho que até melhor para um melhor desenvolvimento.
